specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2021-10-01/mfe.json

```diff
@@ -0,0 +1,10010 @@
+            "$ref": "./examples/OnlineDeployment/K8sOnlineDeployment/createOrUpdate.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/onlineEndpoints/{endpointName}/deployments/{deploymentName}/getLogs": {
+      "post": {
+        "tags": [
+          "OnlineDeployment"
+        ],
+        "summary": "Polls an Endpoint operation.",
+        "operationId": "OnlineDeployments_GetLogs",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "endpointName",
+            "description": "Inference endpoint name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "deploymentName",
+            "description": "The name and identifier for the endpoint.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "The request containing parameters for retrieving logs.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/DeploymentLogsRequest"
+            }
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DeploymentLogs"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Online Deployment Logs.": {
+            "$ref": "./examples/OnlineDeployment/getLogs.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/onlineEndpoints/{endpointName}/deployments/{deploymentName}/skus": {
+      "get": {
```

Comment:
For some reason, I have it in my head that all GET requests MUST respond with a Resource-type object. Is this true? Because GET /skus responds with [DeploymentSkuResourceTypeArmPaginatedResult](https://github.com/Azure/azure-rest-api-specs/blob/326e72400058709b0b15eb7ed4b3a02ee6816498/specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2021-10-01/mfe.json#L6906), which is not an x-ms-azure-resource. @pilor can you double check?

Reply:
In general yes that is true.  Skus have a lot of legacy and don't follow that pattern.  See: https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#enumerating-skus-for-an-existing-resource